modules/apps/data-engine/data-engine-test/src/testFunctional/tests/DEFields.testcase

```diff
@@ -0,0 +1,111 @@
+@component-name = "data-engine"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Data Engine";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			WebContentStructures.tearDownCP();
+		}
+	}
+
+	@description = "LPS-168179 - It's possible to reorder Web Content with Structure of repeatable fields"
+	@priority = 5
+	test ReorderRepeatableFieldsByDragAndDrop {
+		task ("Given a Web Structure that contains repeatable fields") {
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Web Content");
+
+			NavItem.gotoStructures();
+
+			WebContentStructures.addCP(structureName = "WC Structure Title");
+
+			DEBuilder.addField(
+				fieldLabel = "Text field 1",
+				fieldName = "Text",
+				value = "Entry 1");
+
+			Click(
+				key_fieldLabel = "Text field 1",
+				key_fieldName = "Text",
+				locator1 = "FormFields#TEXT_FIELD",
+				value1 = "Text field 1");
+
+			FormViewBuilder.setPlaceholder(fieldValue = "Entry 1");
+
+			Form.gotoAdvancedTab();
+
+			Check.checkToggleSwitch(
+				checkboxName = "Repeatable",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			WaitForElementPresent(locator1 = "Button#PLUS");
```

Comment:
@mateussandes23 Do we also need to wait for the add button to appear?

Reply:
Hey @patriciajeanperez , thanks for again for tips for this PR again. At first sight, the test speed do not allow the poshi to click on dupplicate button, because of that I used this macro to see the button being displayed but now I tested this without the waits and the button appear normaly. I will update this removing this.